README.md

```diff
@@ -1,2 +1,6 @@
 # gh-pr-force-push-test
-Just a test on how force-pushing to a PRed fork branch and later repushing to a different fork affects the PR contents
+
+Just a test on how force-pushing to a PRed fork branch and later
+repushing to a different fork affects the PR contents.
+
+Some changes in here!
```

Comment:
We should be more descriptive about what type of important changes this entails.